test/ginkgo-ext/scopes.go

```diff
@@ -137,6 +137,7 @@ func GinkgoPrint(message string, optionalValues ...interface{}) {
 func GetTestName() string {
 	testDesc := ginkgo.CurrentGinkgoTestDescription()
 	name := strings.Replace(testDesc.FullTestText, " ", "_", -1)
+	name = strings.Trim(name, "*")
```

Comment:
A replace might be more robust (right now the * only happens at the end but it might change). Maybe replacing the `*` with `-` is also ok. 

Alternatively, maybe we can use something like https://golang.org/pkg/net/url/#PathEscape but the output is a little unreadable (maybe only do it on the test name, there is also QueryEscape)